src/options/mkoptions.py

```diff
@@ -228,6 +229,14 @@
   return Options::current()->wasSetByUser(*this);
 }}"""
 
+TPL_IMPL_GET_NAME = \
+"""inline const std::string {name}__option_t::getName() const
```

Comment:
```suggestion
"""const std::string {name}__option_t::getName() const
```

Reply:
`inline` is needed here since it's implemented in the header (outside of the struct)... We should probably change that when we refactor options.

Reply:
Ah right, makes sense.